src/services/mod.rs

```diff
@@ -12,10 +12,12 @@
 //! 5. [C2B Simulate](https://developer.safaricom.co.ke/docs#account-balance-api)
 //! 6. [Mpesa Express/ STK Push](https://developer.safaricom.co.ke/docs#lipa-na-m-pesa-online-payment)
 //! 7. [Transaction Reversal](https://developer.safaricom.co.ke/docs#reversal)
+//! 8. [Bill Manager](https://developer.safaricom.co.ke/Documentation)
```

Comment:
Update the URL, I think you have to link to the BillManager API #...

Reply:
Ahh, Safaricom eventually updated their docs links.